packages/central-server/src/apiV2/index.js

```diff
@@ -242,6 +246,7 @@ apiV2.post('/dashboardRelations', useRouteHandler(CreateDashboardRelation));
 apiV2.post('/dashboardVisualisations', useRouteHandler(CreateDashboardVisualisation));
 apiV2.post('/mapOverlayVisualisations', useRouteHandler(CreateMapOverlayVisualisation));
 apiV2.post('/mapOverlayGroupRelations', useRouteHandler(CreateMapOverlayGroupRelation));
+apiV2.post('/userFavouriteDashboardItem', allowAnyone(handleUserFavouriteDashboardItem));
```

Comment:
```suggestion
apiV2.post('/userFavouriteDashboardItems', allowAnyone(handleUserFavouriteDashboardItem));
```
I think we always use plural, even if it just edits a single record 👍